2d/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ if(build)
         set(VTK_IO_TARGET_LINK_LIBRARIES vtkCommon vtkWidgets vtkIO vtkImaging)
     endif()
 
-    include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include" ${VTK_INCLUDE_DIRECTORIES})
+    include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include")
```

Comment:
Why is this?

Reply:
I don't find related website now again, but as far as I read VTK has since version 6 imported targets, so it is not necessary anymore. And because first compile issues were VTK related I started to play a bit around with it. It is a leftover, but because I don't know if I spend more time on VTK in CMake I didn't removed it before commit.

Reply:
I see. Can we remove this (and similar change in `io`) for now? If somebody (or you) have time in future to confirm that this is not necessary anymore, it would be nice to merge a single PR that fixes all PCL modules, not just one or two.

Reply:
There are no more occurrences of `VTK_INCLUDE_DIRECTORIES` in whole PCL project. But i can remove it from this PR if you want.

Reply:
OK let's keep it here, but would be great if you could put this in a separate commit with an explanatory message.

Reply:
I currently wait for build results to see, if it really works with VTK 6.2, because I'm using 8.2 ;).

Reply:
Moved separate commit into separate PR #2713 ;)